packages/cli/src/lib/strings.ts

```diff
@@ -368,6 +369,11 @@ the PWA:
   promptVersionCode: 'Starting version code for the new app version:',
   promptUpdateProject: 'There are changes in twa-manifest.json. ' +
       'Would you like to apply them to the project before building?',
+  warnFamilyPolicy: bold(red('WARNING: ')) + 'Trusted Web Activities are currently incompatible' +
```

Comment:
Looks good. I wonder if yellow is better for warning, as red can lead developers to think its an error/